src/test/end-to-end/config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  TestUrl: process.env.TEST_URL || 'https://et-sya.aat.platform.hmcts.net',
```

Comment:
please revert to `https://et-sya.aat.platform.hmcts.net`  the preview pipeline will pick up the preview url automatically